release/api/v1alpha1/artifacts.go

```diff
@@ -48,10 +48,6 @@ func (vb *VersionsBundle) Manifests() map[string][]*string {
 		"cilium": {
 			&vb.Cilium.Manifest.URI,
 		},
-		"packages": {
```

Comment:
@ewollesen might have better context for adding this.

Reply:
It could be the build is broken, but we do want to do this